docs/root/configuration/listeners/stats.rst

```diff
@@ -29,6 +29,10 @@ Every listener has a statistics tree rooted at *listener.<address>.* with the fo
    ssl.fail_verify_error, Counter, Total TLS connections that failed CA verification
    ssl.fail_verify_san, Counter, Total TLS connections that failed SAN verification
    ssl.fail_verify_cert_hash, Counter, Total TLS connections that failed certificate pinning verification
+   ssl.ocsp_staple_failed, Counter, Total TLS connections that failed compliance with the OCSP policy
+   ssl.ocsp_staple_omitted, Counter, Total TLS connections that succeeded without stapling an OCSP response
+   ssl.ocsp_staple_responses, Counter, Total TLS connections where the server was capable of stapling an OCSP response
```

Comment:
clarify "was capable of". Does that mean it did staple, or that it could have if the client had requested it, but might not have?